### DISCUSSION TITLE
extra wide screen

### DISCUSSION THREAD

**Discussion by dudeulike** [nicymike/serflings#117] [Show and tell]
CLASSIC game, nice to see a decent port of it. - enjoying my first replay of it right now :)
I accidentally ran it across all 3 monitors - this kind of resolution would have been hilarious back in the amiga days

![threemonitors](https://github.com/nicymike/serflings/assets/116686948/43d7bfb8-7091-4f69-b628-4b7f76ff963d)

**Comment by nicymike** [comment]
Wow, this is crazy. How is the performance eg. FPS (press F3 key).

**Reply by dudeulike** [reply]
Oh, it was poor, but I wasn't surprised by this.
Spanning any program over 3 monitors between 2 different resolutions never goes well :)
I'll load the game up in a bit :)

**Comment by dudeulike** [comment]
I put in a good long session yesterday, 
this is the same game as above, only with a lot more playtime.

![f3info](https://github.com/nicymike/serflings/assets/116686948/c683f120-bb5f-4095-9de4-e2ee89a266f3)

**Reply by dudeulike** [reply]
I'm not actually playing it like this, 
I'm playing it in a window on just one monitor where it's running at 50fps with no issues (capped I'm assuming)
although the traffic and goods routing is a lot more chaotic than I remember it being in the OG version.

**Reply by nicymike** [reply]
Yes capped because game logic runs at 50 ticks per second. Rendering more frames makes no sense because of no updates. 🙂

Traffic and routing should be the same because it is based on the original Assembler code.

**Reply by dudeulike** [reply]
That makes total sense :)
